test/runtests.jl

```diff
@@ -26,3 +26,60 @@ using Test, PerformanceTestTools
         include("test_AbstractInterpreter.jl")
     end
 end
+
+module VSCodeServer
+    using TypedSyntax
+
+    struct InlineDisplay
+        is_repl::Bool
+    end
+    const INLAY_HINTS_ENABLED = Ref(true)
+    const DIAGNOSTICS_ENABLED = Ref(true)
+
+    inlay_hints = []
+    diagnostics = []
+
+    function Base.display(d::InlineDisplay, x)
+        if x isa Dict{String, Vector{TypedSyntax.InlayHint}}
+            push!(inlay_hints, x)
+        elseif eltype(x) == TypedSyntax.Diagnostic
+            push!(diagnostics, x)
+        end
+        return nothing
+    end
+
+    function reset_test_containers()
+        empty!(inlay_hints)
+        empty!(diagnostics)
+    end
+end
+module TestVSCodeExt # stops modules defined in test files from overwriting stuff from previous test
+using Test, PerformanceTestTools, ..VSCodeServer
+@testset "runtests.jl VSCodeExt" begin
+    @testset "test_Cthulhu.jl" begin
+        include("test_Cthulhu.jl")
+    end
+
+    @testset "test_codeview.jl" begin
+        include("test_codeview.jl")
+        include("test_codeview_vscode.jl")
+    end
+
+    # TODO enable this test on nightly
+    if false
+        @testset "test_irshow.jl" begin
+            include("test_irshow.jl")
+        end
+    else
+        @info "skipped test_irshow.jl"
+    end
+
+    @testset "test_terminal.jl" begin
+        include("test_terminal.jl")
+    end
+
+    @testset "test_AbstractInterpreter.jl" begin
+        include("test_AbstractInterpreter.jl")
+    end
+end
+end
```

Comment:
@Zentrik What's the whole purpose of duplicating tests here? Can't we `include("test_codeview_vscode.jl")` only?

Reply:
There's some additional work that Cthulhu does when running under VSCode that I wanted to test in more cases than `"test_codeview_vscode.jl"`. The main thing I was worried about was descending into all callees [here](https://github.com/JuliaDebug/Cthulhu.jl/blob/33c237996f1c67764ce8d80df53e0593fb2ff8b9/src/codeview.jl#L161-L168). 
I don't think it's particularly important to run these additional tests given the code has been working for a while, so if you want you can nix it.

Reply:
So, it sounds like the purpose of running these duplicated tests was to verify that descending into all callees doesn't lead to any timing issues, right? If so, we might consider removing them. Alternatively, if we decide to keep them, it could be a better design choice to extract the main portion of `runtests.jl` into a separate file and then `include` that file in both `runtests.jl` and `test_VSCode.jl`.